pod/perlintro.pod

```diff
@@ -257,12 +282,18 @@ To get at hash elements:
 
  $fruit_color{"apple"};           # gives "red"
 
+Note again the use of the C<$> sigil to access a single hash element.
+The curly braces C<{...}> indicate that it is an hash that we are
+getting the value from.
+
 You can get at lists of keys and values with C<keys()> and
 C<values()>.
 
  my @fruits = keys %fruit_color;
  my @colors = values %fruit_color;
 
+The C<@> sigil was used because we are extracting a list of elements.
```

Comment:
These are actually array variables; maybe an example of a hash slice would be useful just before this comment? like `@fruit_color{"apple","banana"};`

Reply:
How about now?